python_modules/libraries/dagster-dg/dagster_dg/cli/scaffold.py

```diff
@@ -276,23 +390,74 @@ def scaffold_github_actions_command(git_root: Optional[Path], **global_options:
     workflows_dir = git_root / ".github" / "workflows"
     workflows_dir.mkdir(parents=True, exist_ok=True)
 
-    template = SERVERLESS_GITHUB_ACTION_FILE.read_text()
-
     if plus_config and plus_config.organization:
         organization_name = plus_config.organization
         click.echo(f"Using organization name {organization_name} from Dagster Plus config.")
     else:
         organization_name = click.prompt("Dagster Plus organization name") or ""
 
+    if plus_config:
+        gql_client = DagsterPlusGraphQLClient.from_config(plus_config)
+
+        agent_type = _get_deployment_agent_type(gql_client)
```

Comment:
i know we have the 'default' deployment in the config but do we want to let them override it in the CLI? in which case this would maybe need to explicitly pass in the deployment in some way